portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -28,24 +36,52 @@
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
-public class DataServlet extends HttpServlet {
-  private ArrayList<String> trial_array;
-  private String json_array;
-
+public final class DataServlet extends HttpServlet {
+  private ArrayList<Task> tasks;
   @Override
-  public void init() {
-    trial_array = new ArrayList<>();
-    trial_array.add("First hard-coded message");
-    trial_array.add("Second hard-coded message");
-    trial_array.add("Third hard-coded message");
-    json_array = new Gson().toJson(trial_array);
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    String comment = request.getParameter("comment-box");
+    String displayName = request.getParameter("display-name");
+    boolean anonymous = Boolean.parseBoolean(getParameter(request, "anonymous", "false"));
+    long timestamp = System.currentTimeMillis();    
+    
+    if (anonymous) {
+        displayName = "Anonymous";
+    }
+    if (displayName == ""){             //not working correctly right now
```

Comment:
Do you know why it's not working? As a suggestion, I'd put this in a TODO comment above the `if`, like:

`// TODO: display name isn't correctly initialized right now`

Reply:
got rid of it later anyway